### PR TITLE
[SOAR 1787] update error handling to help with trusted host issues

### PR DESCRIPTION
**Pull request by mhofert-r7**
### Description

Describe the proposed changes:

  - add error handling message to indicate that a 401 can be caused by the orchestrator ip not being in the trusted hosts list

### Testing

Any testing information goes here.

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### CODE REVIEW

File: fortinet_fortigate/help.md [hunk 1]
```diff
@@ -646,6 +646,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 4.0.4 - Improve error messaging around 401 return codes to indicate that the orchestrator IP not being in the trusted host list may be the cause
```
Comment (mrinehart-r7) [inline fortinet_fortigate/help.md:649]:
```suggestion
* 4.0.4 - Improve error messaging around HTTP 401 status codes to indicate that the InsightConnect orchestrator IP address not being in the trusted host list may be the cause
```

**Comment by mhofert-r7** [comment]
> Can we document this in briefly in the Setup section and then in more detail in the Troubleshooting section of help.md too so the user doesn't have to wait to see an error to know?

We should use requirements, where it is now, as anything in Setup will be deleted every time you run regenerate

File: fortinet_fortigate/help.md [hunk 2]
```diff
@@ -642,10 +642,12 @@ _This plugin does not contain any triggers._
 
 ## Troubleshooting
 
-_This plugin does not contain any troubleshooting information._
-
+In order for this plugin to function the orchestrator's IP address must be added to the trusted hosts list for the API admin
```
Comment (jschipp-r7) [inline fortinet_fortigate/help.md:645]:
```suggestion
In order for this plugin to communicate with the Fortigate firewall, the orchestrator's IP address must be added to the *trusted hosts* list for the API admin account.
```

File: fortinet_fortigate/help.md [hunk 3]
```diff
@@ -642,10 +642,12 @@ _This plugin does not contain any triggers._
 
 ## Troubleshooting
 
-_This plugin does not contain any troubleshooting information._
-
+In order for this plugin to communicate with the Fortigate firewall, the orchestrator's IP address must be added to the *trusted hosts* list for the API admin account.
+To do this log into the FortiGate firewall. Go to the System tab, Administrator subtab. Then select and edit the API admin.
```
Comment (jschipp-r7) [inline fortinet_fortigate/help.md:646]:
```suggestion
To accomplish this, log into the FortiGate firewall. Go to the System tab -> Administrator subtab and then select and edit the API admin.
```

**Pull request merged**